css/index.css

```diff
@@ -3,144 +3,468 @@
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Roboto', sans-serif;
+html,
+body {
+  height: 100%;
+  font-family: "Roboto", sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-size: 18px;
-    margin-bottom: 15px;
-    font-family: 'Rubik', sans-serif;
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-size: 18px;
+  margin-bottom: 15px;
+  font-family: "Rubik", sans-serif;
 }
 
 p {
-    line-height: 1.4;
+  line-height: 1.4;
+}
+
+/* header  */
+header {
+  width: 100%;
+  background: white;
+}
+
+header nav {
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2% 1%;
+  font-size: 14px;
+  margin: 1% 5%;
+}
+
+.freeway {
+  width: 800px;
+  height: 10vh;
+  /* background-color: purple; */
+  background-image: url("../img/jumbo.jpg");
+  margin-bottom: 2%;
+}
+
+.free {
+  width: 800px;
+  height: 10vh;
+  align-items: center;
+}
+
+.links {
+  width: 68%;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 10%;
+}
+
+header nav a {
+  color: black;
+  text-decoration: none;
 }
 
+.headimg {
+  background-image: url("/img/jumbo.jpg");
+  height: 10vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top center;
+}
+
+.head {
+  background-image: url("../img/jumbo-about.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top center;
+  height: 10vh;
+}
 .container {
-    width: 800px;
-    margin: 0 auto;
+  width: 800px;
+  margin: 0 auto;
+}
+.ab {
+  margin-top: 3%;
+  margin-bottom: 3%;
+}
+
+.bottom {
+  margin-top: 3%;
+  margin-bottom: 3%;
 }
 
```

Comment:
Great job laying out all your content and using flexbox.  Your layouts almost exactly match the design files.  One thing you can do with the header is use space-between this will stretch out the items across the whole header and the beginning image and end anchor will be squared with the banner image.